pkg/model/keycloak_deployment.go

```diff
@@ -417,3 +422,46 @@ func readinessProbe() *v1.Probe {
 		FailureThreshold:    ProbeFailureThreshold,
 	}
 }
+
+func KeycloakPodAffinity(cr *v1alpha1.Keycloak) *v1.Affinity {
+	return &v1.Affinity{
+		PodAffinity: &v1.PodAffinity{
+			PreferredDuringSchedulingIgnoredDuringExecution: []v1.WeightedPodAffinityTerm{
```

Comment:
@ryanemerson Could I ask you to have a look at this? LGTM but having a double check won't hurt ;)